maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/changelog/SvnChangeLogConsumerTest.java

```diff
@@ -39,19 +40,23 @@
 import java.util.TimeZone;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
 /**
  * @author <a href="mailto:evenisse@apache.org">Emmanuel Venisse</a>
  *
  */
 public class SvnChangeLogConsumerTest
-    extends PlexusTestCase
+        extends ScmTestCase
 {
     Logger logger = LoggerFactory.getLogger( getClass() );
 
     SvnChangeLogConsumer consumer;
 
 
-    protected void setUp()
+    public void setUp()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@Before` (in JUnit 4 style)?

Reply:
fixed